### DISCUSSION TITLE
Filebrowser installation on ubuntu server with Apache

### DISCUSSION THREAD

**Discussion by Anaaid** [filebrowser/filebrowser#3124] [Q&A]
Hello everyone !

I am a beginner ^^ 

I'm trying to install filebrowser on ubuntu server with Apache2. 

After downloading the newest version (2.28), I putted files in folder /FBr and here is it's content :
filebrowser (a file without extension, an executable ?)
config.json
filebrowser.db 

After all configurations, I finally got filebrowser launching on my url "https://mydomaine.com/FBr/", but there is some console errors :
 
1 - Failed to load module source from « https://mydomaine.com/FBr/static/assets/index-3ddf7949.js ».
2 - Unable to load « https://mydomaine.com/FBr/static/assets/index-3ddf7949.js ». A service worker intercepted the request and encountered an unexpected error.
3 - The style sheet at https://mydomaine.com/FBr/static/assets/index-14d65c81.css was not loaded because its MIME type, « text/html », is not « text/css ».

I don't understand, I did not find any "static" folder in the "linux-arm64-filebrowser.tar.gz" I downloaded. Do I have to look for this folder elsewhere ? or is that folder in the "filebrowser" file and I have to config a path to it ?

Thank you for your help.

**Comment by gletiecq** [comment]
Filebrowser doesn't need Apache to serve content unless you are using Apache as a reverse proxy - it has a server built-in.  I doubt you need a proxy at this point unless you are trying to have Apache "reverse proxy" for multiple web services (icluding filebrowser) that are served by the same host.  Just follow the instructions for filebrowser and it'll serve web requests without you having to make things more complex than you're trying to make it!
